### PR TITLE
Add CsvUtil class

### LINKED ISSUE #205: Improve code readability for Import Command
Right now `ImoprtCommand` is 225 LoC long with methods that involve reading Csv files that can be extracted away from the execution of `ImportCommand`

### PR DESCRIPTION
**Pull request by Pughal77**
Add CsvUtil

Import Command uses a significant amount of read Csv functions
that i decided to abstract it and put it away in a new utility
class called `CsvUtil`

fixes #205

### CODE REVIEW

**Comment by Zer0Legion** [comment]
Should the export and importExam commands make use of this class?

File: docs/DeveloperGuide.md
Comment (jayllo-c) [inline docs/DeveloperGuide.md:1]: Why does the addition to your DG appear in multiple branches?

File: docs/diagrams/ImportSequenceDiagram.puml
Comment (jayllo-c) [inline docs/diagrams/ImportSequenceDiagram.puml:1]: Why does your diagram appear in multiple PRs?

File: src/main/java/seedu/address/commons/util/CsvUtil.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+/**
+ * Utility class for reading csv files. Used in import Command
+ */
+public class CsvUtil {
+    /**
+     * Reads the csv file and returns a list of maps,
+     * where each map represents a row of person's data in the csv file.
+     * @throws DataLoadingException
+     */
+    public static List<Map<String, String>> readCsvFile(Path filePath) throws DataLoadingException {
```
Comment (jayllo-c) [inline src/main/java/seedu/address/commons/util/CsvUtil.java:26]: Great job simplifying the conversion of csv data into a list

File: src/main/java/seedu/address/logic/commands/ImportCommand.java [hunk 2]
```diff
@@ -83,16 +76,8 @@ public ImportCommand(Path filePath) {
...
-            for (Map<String, String> personData : data) {
-                try {
-                    String addCommandInput = convertToAddCommandInput(personData);
-                    AddCommand addCommand = parseAddCommand(addCommandInput);
-                    addCommand.execute(model);
-                } catch (ParseException e) {
-                    throw new CommandException(String.format(MESSAGE_PARSE_ERROR, personData));
-                }
-            }
+            List<Map<String, String>> personsData = readCsvFile(filePath);
```
Comment (jayllo-c) [inline src/main/java/seedu/address/logic/commands/ImportCommand.java:79]: The abstraction made quite an impact here!

**Review by jayllo-c — COMMENTED** [review]
The CSV util method seems quite handy in abstracting the functionality away from the import command.

**Review by delishad21 — COMMENTED** [review]
Good idea to abstract away CSV related methods into a class. I agree with lengkhai that all import and export related classes should make use of this instead of have their own csv related classes. Well Done!

**Pull request merged**